frontend/src/components/Chat.jsx

```diff
@@ -1,11 +1,32 @@
 import React, { Component } from 'react';
+import axios from 'axios';
 
 import ChatInput from './ChatInput';
 import ChatMessageList from './ChatMessageList';
 
 class Chat extends Component {
   state = {
     messages: [],
+    name: 'Me'
+  }
+
+  // Send message to backend then print it to console
+  sendMessage = (message) => {
+    // add message to chat history
+    this.addMessage(message);
+
+    // send message to backend
+    axios.post('http://0.0.0.0:5555/sentences', {
+        sass: message.text
```

Comment:
this `sass` thing has to change for the future. We should decide on a way to send responses that will be consistent down the line. 

also, i don't like the route name being `sentences`, lets change it to something like `repeat-messages` or `echo`

@svhnstorage @joshcwinton @donaldchen34

Reply:
agreed, i think most things should end up being variations of `message` and `text`

Reply:
created an issue, @svhnstorage @donaldchen34, can one of you pick this up from the project board?